glob/glob.py

```diff
@@ -2,7 +2,15 @@
 
 import os
 import os.path
-import re
+import sys
+
+if (sys.platform == 'uefi') and (sys.implementation.name == 'micropython'):
```

Comment:
Canonical way of doing this is
```
try
  import _re
except ImportError:
  import ure
except importError:
  import re
```
which doesn't have the downside of hardcoding these names.
However, the `import re` was there already: why is it now needed to try the `ure` import (not related to uefi I assume)?

Reply:
Good point regarding re vs ure. I guess my thought was *if* someone were to try and run the micropython under cpython; wouldn't it be better to fall back to standard re which would be available on that platform?

Reply:
If you look in the source code, using the try/except is the canonical way to do this for MicroPython so it is only vague if you haven't seen any of the other code. Though I get what you mean. Still, I don't think we can just go changing these `import re` everywhere. Does it not work for you?